docsets/SDL/docset.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "SDL",
-    "version": "2.0.8",
+    "version": "2.0.10",
     "archive": "org.libsdl.sdl20.docset.tgz",
```

Comment:
This archive does not exist. You added `org.libsdl.sdl20.tgz` instead.

Reply:
Thank you, I fixed the typo.